runtime/common/src/lib.rs

```diff
@@ -172,6 +175,11 @@ pub mod constants {
 	/// Milliseconds per day
 	pub const MILLISECS_PER_DAY: u64 = 86400000;
```

Comment:
Since you added `SECONDS_PER_DAY` etc now, should we redefine this as `SECONDS_PER_DAY * 1000`?

Reply:
Sounds good.